### PR TITLE
Wire v16patch1 & update k8scloudconfig

### PR DESCRIPTION
**Pull request by tuommaki**
Kubernetes v1.11.1 (and also .2 and .3) apiserver has a memory leak.
There is a fix in apimachinery for it, but upstream release PR is
waiting for testing atm. To mitigate this issue for us in the meanwhile,
we built our own patched version of hyperkube image to enable k8s
v1.11.x clusters.

### CODE REVIEW

File: service/controller/v16patch1/version_bundle.go [hunk 1]
```diff
@@ -8,15 +8,20 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "aws-operator",
-				Description: "Made worker node docker volume size to be configurable.",
+				Component:   "calico",
+				Description: "Updated calico to 3.2.0.",
```
Comment (tuommaki) [inline service/controller/v16patch1/version_bundle.go]: This was wrong already in earlier version. Fixed here.

**Comment by r7vme** [comment]
Added NVME changes.

**Comment by tuommaki** [comment]
Created `5.1.0` cluster on `gauss` and upgraded it successfully to `5.1.1`.
Also tested cluster creation with release `5.1.1` on `gauss`. Everything looks good.

All e2e tests also finally passed.

**Comment by tuommaki** [comment]
E2e was green before merge from master and unit tests are green so going with this.

**Pull request merged**